search-app/src/cmr/search/services/parameters/converters/shapefile.clj

```diff
@@ -0,0 +1,146 @@
+(ns cmr.search.services.parameters.converters.shapefile
```

Comment:
This file also uses a lot of Java objects, add type hints where possible here as well.